modules/iam-bootstrap/bootstrap-0.json

```diff
@@ -125,7 +125,8 @@
         "logs:ListTagsLogGroup",
         "logs:ListTagsForResource",
         "logs:PutRetentionPolicy",
-        "logs:TagLogGroup"
+        "logs:TagLogGroup",
+        "logs:TagResource"
```

Comment:
This is fine, but I've been finding that I should've added Untag with a lot of these too, only after seeing an error down the line. Can we add `UntagLogGroup` and `UntagResource`? https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazoncloudwatchlogs.html

Reply:
of course.